src/sections/Meshery/meshery.style.js

```diff
@@ -120,14 +133,42 @@ const MesheryWrapper = styled.section`
         }
     }
 
-    @media only screen and (max-width: 568px) {
+    @media only screen and (max-width: 575px) {
         .info{
+            
             .description {
                 padding: 0;
                 .meshery-hero-img {
                     text-align: center;
                     padding: 0;
-                    min-width: 15rem;
+                    
+                }
+                .desc-text{
+                    text-align: center;
+                    margin-bottom: .75rem;
```

Comment:
You don't need to add same styles again for other media queries until and unless we are restricting the CSS upto a resolution, which we are not in this case. So, please o though the code and remove all duplicate declarations and verify everything looks good.